packages/create-instance/create-component-stubs.js

```diff
@@ -86,31 +86,48 @@ export function createStubFromComponent(
   // ignoreElements does not exist in Vue 2.0.x
   if (Vue.config.ignoredElements) {
     Vue.config.ignoredElements.push(tagName)
+    if (Vue.config.ignoredElements.indexOf('template-stub') === -1) {
```

Comment:
Why is this necessary?

Reply:
So the `template-stub` stays as `<template-stub />` in the finished render, correct?